canonical/ubuntu.com
# WD-7157 - add tables to releases charts

## Done

Charts with release cycles need to be accompanied by the tables with the same data on /about/release-cycle

## QA

- [demo link](https://ubuntu-com-13508.demos.haus/about/release-cycle)
- [copy doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the tables are after each chart
- Check that the data in tables and charts is consistent

## Issue / Card
[WD-7157](https://warthogs.atlassian.net/browse/WD-7157)

Fixes #

## Screenshots


[WD-7157]: https://warthogs.atlassian.net/browse/WD-7157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
